well_known.go

```diff
@@ -8,6 +8,10 @@ import (
 	"strings"
 )
 
+var (
+	errNoWellKnown = errors.New("No .well-known found")
```

Comment:
Could you change this to a `fmt.Errorf`?

Reply:
Not sure I see the benefits of doing that here, but sure I guess?